src/pconn.cc

```diff
@@ -582,22 +584,21 @@ PconnModule::remove(PconnPool *aPool)
 }
 
 void
-PconnModule::dump(StoreEntry *e)
+PconnModule::dump(std::ostream &yaml)
 {
-    typedef Pools::const_iterator PCI;
-    int i = 0; // TODO: Why number pools if they all have names?
-    for (PCI p = pools.begin(); p != pools.end(); ++p, ++i) {
+    for (const auto &p: pools) {
         // TODO: Let each pool dump itself the way it wants to.
-        storeAppendPrintf(e, "\n Pool %d Stats\n", i);
-        (*p)->dumpHist(e);
-        storeAppendPrintf(e, "\n Pool %d Hash Table\n",i);
-        (*p)->dumpHash(e);
+        yaml << "pool " << p->description() << "\n" <<
```

Comment:
Does this line miss a colon? That colon was present in earlier output sample (visible in the current PR description).

```suggestion
        yaml << "pool " << p->description() << ":\n" <<
```